config/ui/ui.go

```diff
@@ -40,6 +42,7 @@ type Config struct {
 	DarkMode        *bool    `yaml:"dark-mode,omitempty"`         // DarkMode is a flag to enable dark mode by default
 	DefaultSortBy   string   `yaml:"default-sort-by,omitempty"`   // DefaultSortBy is the default sort option ('name', 'group', 'health')
 	DefaultFilterBy string   `yaml:"default-filter-by,omitempty"` // DefaultFilterBy is the default filter option ('none', 'failing', 'unstable')
+	DefaultCollapse *bool     `yaml:"default-collapse,omitempty"`  // DefaultCollapse is a flag to enable/disable collapsing of groups by default
```

Comment:
Since this only applies to groups, I think `DefaultGroupCollapsed` would make more sense